addons/mail/static/src/components/message_list/message_list.scss

```diff
@@ -28,6 +28,7 @@
     justify-content: center;
     padding: 20px;
     line-height: 2.5rem;
+    font-style: italic;
```

Comment:
Should be moved to the Style category, not Layout.

I'll test to see how it looks.

Reply:
Would be nice to have an enterprise runbot to test there as well, I think the main "issue" was when the chatter is on the right

Reply:
Done